Cargo.toml

```
@@ -32,7 +32,7 @@ pretty_assertions = "1.3"
 regex = "1"
 reqwest = { version = "0.11", features = ["json"] }
 rnix = "0.11"
-runix = { git = "https://github.com/flox/runix" }
+runix = { git = "https://github.com/flox/runix", branch = "resolve-flakeref-to-gitref" }
```

pending merge upstream 
```suggestion
runix = { git = "https://github.com/flox/runix" }
```